src/index.html

```diff
@@ -15,8 +15,36 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#" class="logo__img"><img src="./images/logo.png" alt="Moyo Logo"></a>
```

Comment:
The `alt` attribute for the logo image should be more descriptive of the image content. Something like 'Company Logo' would be more appropriate than 'Moyo Logo'.